BasePatchGenerator/Program.cs

```diff
@@ -3,36 +3,63 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Net.Mime;
+using System.Security.Cryptography;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace BasePatchGenerator
 {
     class Program
     {
-        static void Main(string[] args)
+	    private static void Main(string[] args)
         {
-            FileStream fs = new FileStream("zelda.sfc", FileMode.Open, FileAccess.Read);
-            byte[] rom_data = new byte[fs.Length];
+	        if (args.Length != 3)
+	        {
+				Console.WriteLine("Usage: BasePatchGenerator.exe [rom file] [patch data] [output file]");
+				return;
+	        }
```

Comment:
The codebase seems to use 4-space indents, generally. I'd appreciate it if you converted away from tabs when commiting. Idc if you like to use tabs, it's all good, I just want to keep the code consistent.

Perhaps an .editorconfig might be a nice addition? If you think so, I'll make one.

Reply:
Could totally use a .editorconfig.

Reply:
I'll whip one up for future inclusion.